internal/commands/push/command_test.go

```diff
@@ -1429,6 +1431,89 @@ func TestPushCommandDeployDraftAndWait(t *testing.T) {
 	})
 }
 
+func TestPushCommandImportDependenciesAndWait(t *testing.T) {
+	groupID, appID, uploadPath := "groupID", "appID", "node_modules.zip"
+	testError := errors.New("something bad happened")
```

Comment:
other tests just kept re-introducing this variable which I thought was a bit, I'm not sure if this is better and can just do it like the rest if that's better for readability.

Reply:
this works for me.

Reply:
personally not a huge fan of this, but its not a big deal either way.  in my mind, its just one more hop when reading test code to figure out what exactly is being asserted... in generally DRYing up test code often leads to just more complexity and difficulty to read